cmd/neo-exporter/config.go

```diff
@@ -33,7 +33,7 @@ const (
 
 func DefaultConfiguration(cfg *viper.Viper) {
 	cfg.SetDefault(prefix+delimiter+cfgNeoRPCEndpoint, "")
-	cfg.SetDefault(prefix+delimiter+cfgNeoRPCDialTimeout, 5*time.Second)
+	cfg.SetDefault(prefix+delimiter+cfgNeoRPCDialTimeout, time.Minute)
```

Comment:
Is it documented anywhere?

Reply:
I don't understand what you mean. Do I need to leave a comment in the code?

Reply:
I mean some user-exposed documentation. Looks like we don't have it here, just a sample config file. OK then, nothing to change.